.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       matrix:
         os:
           - "ubuntu-22.04-8core"
-          - "macos-latest-xl"
+          - "macos-14-xlarge"
```

Comment:
Pulled out of the auto-merge queue because I've just realised while debugging https://github.com/flox/flox/pull/1687 that the platforms are hardcoded into the cross-system test. Will update and re-request review tomorrow. I suspect it will require some a few runs again to get both co-dependent jobs passing.

Reply:
You can now see it passing here: https://github.com/flox/flox/actions/runs/9659972365/job/26644497524#step:8:73